app/plugins/misc/roast.py

```
@@ -0,0 +1,42 @@
+
+
+@BOT.add_cmd(cmd="roast")
+async def roast(bot: BOT, message: Message):
+    """
+    CMD: ROAST
+    INFO: Deliver a random roast to a user.
+    USAGE: .roast <reply/username>
+    """
+    user, _ = await extract_user(bot, message)
+    if not user:
+        await reply_and_delete(message, f"I can't find that user.")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)